### PR TITLE
hint: add force priority hint for select

### PR DESCRIPTION
**Pull request by leoppro**
add force priority hint for select, users can control it by tidb-force-priority.

### CODE REVIEW

**Comment by leoppro** [comment]
PTAL @GregoryIan @csuzhangxc

**Comment by GregoryIan** [comment]
@kennytm PTAL

File: mydumper.c [hunk 1]
```diff
@@ -1916,7 +1936,7 @@ GList * get_chunks_for_table(MYSQL *conn, char *database, char *table, struct co
 	if (!field) goto cleanup;
 
 	/* Get minimum/maximum */
-	mysql_query(conn, query=g_strdup_printf("SELECT %s MIN(`%s`),MAX(`%s`) FROM `%s`.`%s`", (detected_server == SERVER_TYPE_MYSQL) ? "/*!40001 SQL_NO_CACHE */" : "", field, field, database, table));
+	mysql_query(conn, query=g_strdup_printf("SELECT %s MIN(`%s`),MAX(`%s`) FROM `%s`.`%s`", select_hint, field, field, database, table));
```
[the remark this answers is not in the document]
Comment (kennytm) [reply mydumper.c]: https://en.wikipedia.org/wiki/Drizzle_%28database_server%29

> Drizzle is an **abandoned** free software/open-source relational database management system (DBMS) that was forked from the now-defunct 6.0 development branch of the MySQL DBMS.

The last release of Drizzle is 6 years ago. The official website is not reachable. Given that (1) this software has nothing to do with PingCAP and (2) it is abandoned, I think it's safe to drop support of Drizzle.

[the remark this answers is not in the document]
Comment (morgo) [reply mydumper.c]: Yes, I agree.  Mydumper could be improved if it removed support for Drizzle and very old MySQL servers.  MySQL 5.6 is the minimum officially supported MySQL, and it has the query cache disabled for instance.  MySQL 8.0 completely removes it.

Some of these decisions may be better made by upstream, but I think removing drizzle is very safe.

**Pull request merged**